BrainPortal/app/controllers/neurohub_portal_controller.rb

```diff
@@ -55,5 +55,47 @@ def search
     @projects = report[:projects]
   end
 
+  def nh_show_license #:nodoc:
+
+    @license = params[:license].gsub(/[^\w\/-]+/, "")
+
+    render :show_infolicense if @license&.end_with? "_info" # info license does not require to accept it
+  end
+
+  def nh_sign_license #:nodoc:
```

Comment:
We also have duplication here with `portal_controller.rb` not sure if we can avoid it.

Reply:
Me too. I think neurohub mostly follows cbrain duplication philosophy, rather reuse to allow for greater independence I guess.

We can create a module just for show_license` ? Is it worth for 2 line method? If we did not follow the convention to prefix actions with nh, the move would be a bit easier, and `sign_license` could be moved too

Reply:
is not neurohub a bit more descriptive than cryptic nh abbreviation? I can try of course and keep my finger crossed

Reply:
I did as requested but as per Pierre recommendation rolled back

Reply:
to neurohub